runtime/compiler/env/j9methodServer.hpp

```diff
@@ -45,6 +45,27 @@ TR_J9MethodFieldAttributes
    bool unresolvedInCP;
    bool result;
    };
+struct
+TR_J9MethodResolvedRelocatableFieldAttributes
+   {
+   // for field attributes, use fieldOffset,
+   // for static attributes, use address
+   union
+      {
+      IDATA fieldOffset;
+      void *address;
+      };
+   bool unresolvedInCP;
```

Comment:
Please move `unresolvedInCP` after `ltype`, otherwise the compiler will insert padding and increase the size of the struct unnecessarily.